packages/convergence/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Changed
+
+- updated inline api docs
```

Comment:
- Added docs generator (using JSDoc)

Reply:
The docs generator is more of an addition to the entire repo. Unless I should add it for each changelog along with [this commit](https://github.com/thefrontside/bigtest/pull/86/commits/83ec89a231df0a41faedd5ad0f1a91b73d4c437a)?

Reply:
Hm, that's an interesting point. I'm not sure how to handle that. 🤷‍♂️ mono repos

Reply:
I'll add

```
- added docs script
```

to each one. Since the generator itself is a repo addition, but the script entries are per package changes.

Reply:
Added